micro-rdk/src/esp32/entry.rs

```diff
@@ -114,7 +132,7 @@ pub async fn serve_web_inner(
             }
         };
 
-        (cfg_response, robot)
+        (cfg_response, robot, tls_server_config)
```

Comment:
I've always wondered this but forgot to ask; why do we need this TLS config to still be in scope for the rest of this function?

Reply:
it is used buy the http2 server if configured in the Viam Server Build